hyperlane-mailbox/src/interface.sw

```diff
@@ -25,3 +30,15 @@ abi Mailbox {
     #[storage(read)]
     fn latest_checkpoint() -> (b256, u32);
 }
+
+abi MessageRecipient {
+    #[storage(read,write)]
+    fn handle(origin: u32, sender: b256, message_body: Vec<u8>);
```

Comment:
I don't feel strongly here because I can move over to Bytes in https://github.com/hyperlane-xyz/fuel-contracts/issues/22, but if along the way when writing tests this proves easier to be `Bytes`, feel free to change